### ISSUE TITLE
[Form and Labels-Howdy-for-pms]: Label is not provided for search text box in the howdy for pms page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#19]
**User Experience**
Screen reader user will not know what to enter in the edit box if it doesn't have the associated label to it.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://hcl-szz1925.howdy.ai/dashboard
Browser: Edge
 
**Repro Steps**
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Navigate to search text box and open the developer tool bar by using F12 key.
Step 6: Observe the issue 

**Actual Result**
Label is not provided for search text box in the howdy for pms page.

**Expected Result**
Label should be provided for search text box in the Howdy for pms page.

**Suggested Fix**
aria-label attribute does not exist or is empty
aria-labelled by attribute does not exist, references elements that do not exist or references elements that are empty
Form element does not have an implicit (wrapped) <label>
Form element does not have an explicit <label>
Element has no title attribute, or the title attribute is empty

**Mas Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}
![MAS3 3 2_Form labels-SEARCH](https://user-images.githubusercontent.com/48272005/54473987-33a50580-4805-11e9-9631-d957a2bb65a1.png)
![MAS3 3 2_Form labels](https://user-images.githubusercontent.com/48272005/54473990-37d12300-4805-11e9-9fad-3e0208fd04c7.png)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
